Update examples with type and API version

sam-cogan opened the pull request:
Current examples do not compile as they are missing the type and API Version properties, added these in.

azurecla commented:
Hi __@MSFTMan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (deonhe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

sam-cogan committed:
Update examples with type and API version

Current examples do not compile as they are missing the type and API Version properties, added these in.
